build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.0.2'
+        classpath 'com.android.tools.build:gradle:4.1.1'
```

Comment:
Any specific reason to degrade version? I mean.. I don't prefer to degrade versions of depenencies unless it causes a problem

Reply:
Android Studio did not want to sync a project on my PC. I'll remove this change.